actors/miner/src/deadline_info.rs

```diff
@@ -85,6 +86,24 @@ impl DeadlineInfo {
         }
     }
 
+    pub fn new_from_policy(
```

Comment:
I'm not a fan of having a new method in the DeadlineInfo impl just for testing. Can this instead be in the test file itself?

Reply:
This looks like a duplicate of `new_deadline_info` from actors/miner/src/deadlines.rs.

Reply:
Please use that one instead before merging